### ISSUE TITLE
Thanking you

### ISSUE THREAD

**Issue by anthony-geoghegan** [docwhat/itsalltext#95]
This is probably an abuse of GitHub's *Issues* feature but since this project is no longer being maintained, I figured the nice/right thing to do is to publicly thank **docwhat** for putting the time and effort into making this useful Firefox extension and for continuing to maintain it so that it has been freely available to so many of us.

*This post has been edited using the ItsAllText extension.*

**Comment by keryell** [comment]
I have been using it for so long that I cannot really remember when I started using it for my Emacs on a daily basis....

This saved my life and is still doing so (but for how long? Suspense...)

Thank you so much!

**Comment by quodlibetor** [comment]
Agreed, it's all text has been an essential part of my workflow for so many years I don't even know when I started. Thanks @docwhat !

**Comment by scoopex** [comment]
Thank you.

**Comment by Li-An** [comment]
Thanks a lot. I switched to Waterfox mainly because your extension is so convenient for me.

**Comment by twirrim** [comment]
I've been using "It's All Text" for several years now, after a co-worker suggested it, and has been a staple part of my workflow ever since.  Thanks for all your hard work 👍

**Comment by docwhat** [comment]
😊

**Comment by sputnick-dev** [comment]
Same thing, I use it since so many year, I think something like 10 years, every day.

Very handy to rearrange lines with `vim` !
